modules/tinymce/src/core/main/ts/api/html/Schema.ts

```diff
@@ -925,7 +925,7 @@ function Schema(settings?: SchemaSettings): Schema {
   const getShortEndedElements = (): SchemaMap => shortEndedElementsMap;
 
   /**
-   * Returns a map with self closing tags such as <li>.
+   * Returns a map with self closing tags such as LI.
```

Comment:
```suggestion
   * Returns a map with self closing tags such as <code>&#60;li&#62;</code>.
```
?

Reply:
Do we care that the lines further up use all caps element names?

Reply:
Discussed offline, have changed all HTML references in this file to match Tyler's suggestion.